library/zlib/deflate/deflate_spec.rb

```diff
@@ -60,6 +60,12 @@
   end
 end
 
+describe "Zlib#deflate" do
+  it "handles call on a module" do
+    Zlib.deflate("1" * 10).should == [120, 156, 51, 52, 132, 1, 0, 10, 145, 1, 235].pack('C*')
+  end
+end
```

Comment:
This spec should be in the (new) file ` library/zlib/deflate_spec.rb` to follow the [file organization](https://github.com/ruby/spec/blob/master/CONTRIBUTING.md#file-organization), could you move it there?
The same applies for Zlib#inflate below.

Reply:
sure, but there's an inconsistency. For example at this line
https://github.com/ruby/spec/blob/master/library/zlib/deflate/deflate_spec.rb#L28
we use Zlib.deflate instead of Zlib::Deflate.deflate, is it intentional?

Reply:
That's a mistake indeed.
I moved it in 7753d2f290fad93a92f02fc9c8a8d1c07123cebd without noticing and the [original commit](763d9eaadd9ac839d7aa16f9b0ffd657b07c1beb) should have used 2 different files.
I'll fix those, thanks for noticing.

Reply:
Fixed in a1b0e473f19eabfad98e6b7e4530a61ca9729ff3